### PR TITLE
[SNDVOL32] Small Italian translation update

### PR DESCRIPTION
**Pull request by danielecampione**
[SNDVOL32] Better Italian translation.

## Purpose
Better Italian translation.

## Proposed changes
I want to enhance the Italian translation.

### CODE REVIEW

File: base/applications/sndvol32/lang/it-IT.rc [hunk 1]
```diff
@@ -113,24 +113,24 @@ END
...
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL IDI_MAINAPP, -1, "static", SS_ICON | WS_CHILD | WS_VISIBLE, 7, 7, 32, 32
-    LTEXT "These settings can be used to make fine adjustments to your audio.", -1, 44, 7, 200, 32
-    GROUPBOX "Tone Controls", IDC_ADV_TONE_CONTROLS, 7, 33, 240, 80
-    LTEXT "These settings control how the tone of your audio sounds.", -1, 17, 48, 200, 14
+    LTEXT "Queste impostazioni possono essere usate per fare regolazioni fini al tuo audio.", -1, 44, 7, 200, 32
+    GROUPBOX "Controlli di tono", IDC_ADV_TONE_CONTROLS, 7, 33, 240, 80
+    LTEXT "Queste impostazioni controllano il modo in cui suona il tono del tuo audio.", -1, 17, 48, 200, 14
     LTEXT "&Bass:", -1, 17, 62, 50, 8
```
Comment (binarymaster) [inline base/applications/sndvol32/lang/it-IT.rc]: Maybe these words would be more appropriate:

**Basso**: _Quiete_ / _Louder_
**Alto**: _Quiete_ / _Louder_

Currently you have:

Bass: **Basso** / **Alto**
Treble: **Basso** / **Alto**

Cc @Fraizeraust and @ElCresh

Comment (Fraizeraust) [reply base/applications/sndvol32/lang/it-IT.rc]: For **Bass** and **Treble** the author could just use `Basso` for Bass and `Alto` for Treble (with further indicating that these are range tones and not related with High/Low).

Comment (ElCresh) [reply base/applications/sndvol32/lang/it-IT.rc]: In italian we indicate freq with "Alti" (plural!) (Trebble) and "Bassi" (plural) (Bass) and the volume with "Basso" (singular) and "Alto" (singular). So a mix of the two is the right answers.

**Comment by danielecampione** [comment]
Yes, you're right.
I will change "Trebble" with "Alti" and "Bass" with "Bassi", as requested.
Thanks.

**Comment by danielecampione** [comment]
> 
> 
> Yes, you're right.
> I will change "Trebble" with "Alti" and "Bass" with "Bassi", as requested.
> Thanks.

I've created the Pull request:
https://github.com/reactos/reactos/pull/1431

**Pull request closed without merging**